app/controllers/medicaid/income_job_number_controller.rb

```diff
@@ -4,6 +4,22 @@ module Medicaid
   class IncomeJobNumberController < MedicaidStepsController
     private
 
+    def existing_attributes
+      HashWithIndifferentAccess.new(number_of_job_attributes)
+    end
+
+    def number_of_job_attributes
+      { new_number_of_jobs: number_of_jobs }
+    end
+
+    def number_of_jobs
+      if current_application.new_number_of_jobs&. > 4
```

Comment:
`current_application.new_number_of_jobs&. > 4`

Yoooooo. Lonely operator on the `>`?? Look at you PFFT

![](https://m.popkey.co/cea493/mxXX_f-maxage-0_s-200x150.gif)

Reply:
oh yeah I actually got an error in my specs when I didnt have this!